packages/components/src/components/heading/heading-styles.ts

```diff
@@ -0,0 +1,91 @@
+import type { JssStyle } from 'jss';
+import type { HeadingVariant, TextAlign, TextColor, TextSize, Theme, VariantType } from '../../types';
+import { buildResponsiveStyles, getCss, textMap } from '../../utils';
+import { addImportantToEachRule, getThemedColors } from '../../styles';
+import {
+  headingLargeStyle,
+  headingXLargeStyle,
+  headingMediumStyle,
+  headingSmallStyle,
+  headingXXLargeStyle,
+  fontWeight,
+  textSmallStyle,
+  displayLargeStyle,
+} from '@porsche-design-system/utilities-v2';
+import { getEllipsisJssStyle, getSlottedTypographyJssStyle } from '../../styles/typography-styles';
+import { isVariantType } from './heading-utils';
+
+const headingMap: Record<VariantType, JssStyle> = {
+  'large-title': displayLargeStyle,
```

Comment:
Can't we use `HEADING_VARIANTS` here somehow instead of redefining everything?

Reply:
we don't want to use variants at all for the new component.